process_labels/loading_funcs/tanzania_ecaas.py

```diff
@@ -0,0 +1,84 @@
+from pathlib import Path
+import json
+import geopandas
+import pandas as pd
+from datetime import datetime
+from shapely.geometry import Point
+from cropharvest.columns import RequiredColumns, NullableColumns
+from cropharvest.config import EXPORT_END_MONTH, EXPORT_END_DAY
+
+# from ..utils import DATASET_PATH # ImportError: attempted relative import with no known parent package
+
+from typing import List
+
+LABEL_TO_CLASSIFICATION = {
+    "rice": "cereals",
+    "maize": "cereals",
+}
+
+
+def load_tanzania_ecaas():
+
+    # Field_files = (DATASET_PATH / "tanzania/tanzania_rice_ecaas").glob("*.csv")
+    file_path = Path("process_labels/raw_data/tanzania/tanzania_rice_ecaas")
```

Comment:
How exactly were you running the code? `DATASET_PATH` should work, and has the advantage of being invariant to how / where the code is run from (this implementation breaks depending on how the function is called).

Reply:
I was calling the `tanzania_ecaas.py` directly. But I've seen how I should call it now. 
Works fine now.